File: test/test-assign/rmd-correct/student/tests/q3.R
test = list(
  name = "q3",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.6666666666666666,
      code = {
        test_that("q3a", {
            expect_equal(nine, 9)
        })
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.6666666666666666,
      code = {
        test_that("q3b", {
            expect_equal(square(16), 256)
        })
      }
    )

  )
)

File: test/test-notebook/tests/q3.py
OK_FORMAT = True

test = {
	"name": "q3",
	"points": 1,
	"suites": [ 
		{
			"cases": [ 
				{
					"code": r"""
					>>> import tqdm
					>>> print(tqdm.__name__)
					tqdm
					>>> 1 == 1
					True
					""",
					"hidden": False,
				}, {
					"code": r"""
					>>> import tqdm
					>>> print(tqdm.__name__)
					tqdm
					>>> 1 == 1
					True
					""",
										"locked": False,
				}
			],
			"scored": False,
			"setup": "",
			"teardown": "",
			"type": "doctest"
		}, 
	]
}
